R/makeTreeSummarizedExperimentFromBiom.R

```diff
@@ -142,6 +142,14 @@ makeTreeSEFromBiom <- function(
         # Add correct colnames
         colnames(feature_data) <- colnames
     }
+    # If there is only one column in the feature data, it is the most probable
+    # that the taxonomy is not parsed. Try to parse it.
+    if( ncol(feature_data) == 1 ){
+        tax_tab <- .parse_taxonomy(feature_data, column_name = colnames(feature_data))
+        feature_data <- cbind(tax_tab, feature_data)
+        feature_data <- as.data.frame(feature_data)
+    }
+    
```

Comment:
I encountered BIOM files that have only one column named "taxonomy". It included taxonomy info in a format "k_bacteria;p__something"